docs-gluu-server-prod/docs/source/operation/logs.md

```diff
@@ -281,6 +281,47 @@ Remember the initial `GET` request will hit the Apache server first, and then be
 
 ## Clearing Logs
 
+### VM setup
+
 The "clear-logs" feature can be used to clear all log files with the extension `.log`, including system logs. Clear-logs can be helpful during troubleshooting, as well as research and development. We discourage its use in production systems if the logs are not backed-up. More features are planned. Please consider this as work in progress.
```

Comment:
Hi @misba7 

`Please consider this as work in progress`

Is this still WIP?

Reply:
Maybe we can ask @moabu ?

Reply:
To me this is operational and yes it should stay WIP until the next release